app/controllers/activities_controller.rb

```diff
@@ -0,0 +1,12 @@
+class ActivitiesController < ApplicationController
+	def home
+		@user = current_user
+		@activities = @user.activity_feed.order(created_at: :desc)
+		@activity = @user.activities.where(action_type: "Lesson")
```

Comment:
@saki4567 You can use this sentence for readability, since what you want to count is lessons:
`@lessons =  @user.lessons`